kalliope/brain.yml

```diff
@@ -1,18 +1,10 @@
 ---
-  - includes:
-    - brains/ansible_playbook.yml
-    - brains/gmail_checker.yml
-    - brains/kill_switch.yml
-    - brains/openweathermap.yml
-    - brains/push_message.yml
-    - brains/rss_reader.yml
-    - brains/say.yml
-    - brains/script.yml
-    - brains/shell.yml
-    - brains/systemdate.yml
-    - brains/tasker_autoremote.yml
-    - brains/wake_on_lan.yml
-    - brains/twitter.yml
-    - brains/neurotransmitter.yml
-    - brains/wikipedia.yml
-    - brains/default.yml
+  - name: "Say-hello"
+    signals:
+      - order: "bonjour"
+      - order: "Bonjour"
+      - order: "hello"
+    neurons:
+      - say:
+          message:
+            - "Bonjour monsieur"
```

Comment:
Answer in english.
Maybe should make 2 synapses : one in french another in english ?